rapid7/metasploit-framework
# Update to Rails 5.2

Upgrade to Rails 5.2.

Brings the framework environment up to an officially supported version of Rails.

## Verification

* Start `msfconsole` with various database states
- [x] no database
- [x] database configured via `msfdb init`
- [x] database config in `~/.msf4/database.yml` based on `config/database.yml.example`
- [x] database config in `config/database.yml` based on `config/database.yml.example`

* **Verify** database commands function as expected
- [x] `hosts`
- [x] `vulns`
- [x] `db_import`
- [x] `db_nmap`

* **Verify** modules function as expected
- [x] tester selected module 1
- [x] tester selected module 2
- [x] tester selected module 3
- [x] tester selected module 4

* **Verify** sessions interactions
- [x] windows meterpreter session (created with psexec)
- [ ] windows shell session (created with psexec)
- [x] linux meterpreter session (created with `msfvenom`)
